get_altitudes.cpp

```diff
@@ -79,6 +80,7 @@ PLUGIN_API int XPluginStart(char* outName, char* outSig, char* outDesc) {
     // Obtain datarefs for MSL and AGL elevation, respectively
     elevationMslRef = XPLMFindDataRef("sim/flightmodel/position/elevation");
```

Comment:
We need to figure out exactly what DataRefs we need to have in our code. We are confused and going to ask the client about this when we meet this week.

Reply:
I would wait to merge this until we receive confirmation about these DataRefs.